### PR TITLE
Simplify llnl import

### LINKED ISSUE #1359: IMAGE PLATE 3 for TARDIS import
During the loading of the image plate 3 for the TARDIS import tool, the boundaries of the image are correctly defined, but part of the image plate does not have usable intensity (the part where the intensity suddenly drops at the edges). This part of the image plate is bleeding out during the stereographic projection as well as the polar view. 

The boundaries should remain unchanged so that the user is able to line up the boundaries with the image, but when the user hit `complete and reload detectors` then, only the middle part of the image should be selected. The image below shows the part of the image plate 3 that needs to be cropped off. Equal amounts should be cropped from left and right (equal to the cropping on the right side since it's larger)

Picture1

### LINKED ISSUE #1363: Automatically read TARDIS hdf and automatically flip images.
TARDIS hdf files store the data in a similar way, ie, XXX/DATA/DATA. It would save clicks if we could just have that read automatically. 

I attached a compressed python method that I use in other code that may be a good starting place. Not all of the if statements are necessary for TARDIS.

[load_nif_im_from_h5.py.zip](https://github.com/HEXRD/hexrdgui/files/11083683/load_nif_im_from_h5.py.zip)

### LINKED ISSUE #1374: Draw the detector outlines in TARDIS import by default
It's an extra click to do this, and I do it every time.

### PR DESCRIPTION
**Pull request by bnmajor**
This adds a handful of features to simplify the LLNL workflow:

- The template is automatically applied when a detector is selected and an image is loaded (Fixes #1374)
- The cropped IP3 boundary uses the dashed line instead of a solid one (Fixes #1359)
- Automatically prompt user to apply the default transform associated with the current instrument/detector (Fixes #1363)
- Adds a help button to the documentation

### CODE REVIEW

File: hexrd/ui/llnl_import_tool_dialog.py [hunk 1]
```diff
@@ -138,6 +141,11 @@ def load_config(self):
         self.config_file = selected_file if selected_file else None
         self.ui.config_file_label.setText(os.path.basename(self.config_file))
         self.ui.config_file_label.setToolTip(self.config_file)
+        if self.ui.instrument.isEnabled():
+            # Only set the instrument config once
+            success = self.get_instrument_defaults()
+            if not success:
+                return
```
Comment (psavery) [inline hexrd/ui/llnl_import_tool_dialog.py:148]: Is there supposed to be different behavior depending on the value of `success`? It looks like this function returns anyways after this block ends.

File: hexrd/ui/llnl_import_tool_dialog.py [hunk 2]
```diff
@@ -177,6 +186,11 @@ def update_config_selection(self, checked):
         elif not self.ui.default_config.isChecked():
             self.enable_widgets(self.ui.load_config, self.ui.config_file_label,
                                 enabled=True)
+        if self.ui.instrument.isEnabled():
+            # Only set the instrument config once
+            success = self.get_instrument_defaults()
+            if not success:
+                return
```
Comment (psavery) [inline hexrd/ui/llnl_import_tool_dialog.py:193]: Same here

File: hexrd/ui/llnl_import_tool_dialog.py (hunk 1 above)
Comment (bnmajor) [reply hexrd/ui/llnl_import_tool_dialog.py]: Nope, careless copy-paste. Fixed.

File: hexrd/ui/llnl_import_tool_dialog.py (hunk 2 above)
Comment (bnmajor) [reply hexrd/ui/llnl_import_tool_dialog.py]: Same careless copy-paste. Fixed.

**Review by psavery — APPROVED** [review]
LGTM. We will keep improving this in further iterations in the near future.

**Pull request merged**